modules/learn/pages/clusters-and-availability/node-to-node-encryption.adoc

```diff
@@ -27,14 +30,20 @@ Node-to-node encryption is disabled by default.
 Note that auto-failover must be switched off both for a change to be made to the cluster’s address-family, and for encryption to be enabled or disabled: auto-failover can be switched back on after the necessary changes have been made.
 (This requirement prevents nodes that are undergoing changes from being determined unresponsive during the process, and unnecessarily failed over.)
 
+Note also that in all versions of Couchbase Server that are 6.6 or later, all Eventing functions must be _paused_ before node-to-node encryption for the cluster is enabled, disabled, or is established at a new level: this prevents loss of mutations.
```

Comment:
should be 6.6.5?

Reply:
My current understanding is that function must be paused and resumed irrespective of whether the Eventing Service itself is supported by node2node encryption. But is that perhaps not the case?